openmetadata-ui/src/main/resources/ui/src/locale/languages/de-de.json

```diff
@@ -15,6 +15,7 @@
     "acknowledged": "Acknowledged",
     "action": "Action",
     "action-plural": "Aktionen",
+    "action-required": "Action Required",
```

Comment:
can we add translations for the respective language?

Reply:
added translations